cn/docs/tasks/run-application/deployment-patch-demo.yaml

```diff
@@ -1,9 +1,12 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1beta2
```

Comment:
@chenopis @steveperry-53, do we typically edit the `cn/docs/...` YAMLs directly?

Reply:
@markthink, @zhangxiaoyu-zidif, Can you answer this question about editing the cn files directly?

Reply:
I contacted with @markthink, he said it's fine to modify any .yaml files in /cn.
I think so.
@steveperry-53 @foxish